RxCocoa/iOS/UICollectionView+Rx.swift

```diff
@@ -179,6 +179,8 @@ extension UICollectionView {
 }
 
 extension Reactive where Base: UICollectionView {
+    public typealias DisplayCollectionViewCellEvent = (cell: UICollectionViewCell, indexPath: IndexPath)
```

Comment:
I think this should be `.. at: IndexPath)`.
I understand that we currently have `indexPath` in other places in this project, but it seems to me that it would be nice to change this in 4.0, so why not start now.